GitHubRepositories.ps1

```diff
@@ -68,6 +68,12 @@ filter New-GitHubRepository
     .PARAMETER NoWiki
         By default, this repository will have a Wiki.  Specify this to disable the Wiki.
 
+    .PARAMETER NoPages
```

Comment:
We should try to keep PR's as atomic as possible (which helps in cases like this where one part of the change is desired, but an unrelated change is contentious).  If you split this change out, we can likely get it merged in faster.

Reply:
Reverted